src/redux/features/items/itemsSlice.js

```diff
@@ -114,18 +114,18 @@ const itemsSlice = createSlice({
       //console.log('current shop items in updateShopItem   =>', current(state).shopItems);
```

Comment:
commented console log 👀

Reply:
It was more for me to understand what i was doing when learning redux and how to use.